core/handler/gear_action/web.ex

```diff
@@ -141,7 +141,8 @@ defmodule AntikytheraCore.Handler.GearAction.Web do
            entry_point :: v[GearEntryPoint.t()],
            helper_modules :: v[HelperModules.t()]
          ) :: Conn.t() do
-    ExecutorPoolHelper.with_executor(conn1, gear_name, helper_modules, fn pid, conn2 ->
+    ExecutorPoolHelper.with_executor(conn1, gear_name, helper_modules, fn _pid, conn2, epool_id ->
+      {:ok, pid} = ActionRunner.start_link(epool_id)
```

Comment:
IIUC, this process won't be killed.

Reply:
Thank you! The process should be killed after working in order to prevent an infinite increase of processes.
Fixed at https://github.com/access-company/antikythera/pull/162/commits/2d5957fb7f35988ebd34067a12ce0d040401f25f